corehq/apps/locations/management/commands/migrate_new_location_fixture.py

```diff
@@ -0,0 +1,40 @@
+import json
+from django.core.management.base import BaseCommand
+from toggle.models import Toggle
+
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.domain.models import Domain
+from corehq.toggles import HIERARCHICAL_LOCATION_FIXTURE, NAMESPACE_DOMAIN
+
+
+class Command(BaseCommand):
+    help = """
+    To migrate to new flat fixture for locations. Enable FF HIERARCHICAL_LOCATION_FIXTURE for
+    apps with locations and having commtrack:enabled in app files
+    The Feature Flag FLAT_LOCATION_FIXTURE should be removed after this
+    """
+
+    def handle(self, *args, **options):
+        domains_having_locations = (
+            SQLLocation.objects.order_by('domain').distinct('domain')
+            .values_list('domain', flat=True)
+        )
+        domains_with_hierarchical_fixture = find_applications_with_hierarchical_fixture(
+            domains_having_locations
+        )
+        toggle = Toggle.get(HIERARCHICAL_LOCATION_FIXTURE.slug)
+        for domain in domains_with_hierarchical_fixture:
+            toggle.set(domain, True, NAMESPACE_DOMAIN)
+
+
+def find_applications_with_hierarchical_fixture(domains):
+    search_string = 'commtrack:locations'
+    domain_with_application = {}
+    for domain in domains:
+        domain_obj = Domain.get_by_name(domain)
+        for application in domain_obj.applications():
```

Comment:
is there a chance we think we need to search app builds too? maybe just released ones? failure mode I'm thinking of is if someone has removed references in the current app version but still has users using earlier app versions.